modules/core/include/opencv2/core/quaternion.inl.hpp

```diff
@@ -846,4 +869,4 @@ Quat<T> Quat<T>::spline(const Quat<T> &q0, const Quat<T> &q1, const Quat<T> &q2,
 }  // namepsace
 //! @endcond
 
-#endif /*OPENCV_CORE_QUATERNION_INL_HPP*/
+#endif /*OPENCV_CORE_QUATERNION_INL_HPP*/
```

Comment:
please keep single empty "new line"

Reply:
Do you mean to create a new empty line at the end？

Reply:
Yes, files should have with single "new line" symbol in the end.